modules/mod_admin_statistics/templates/admin_statistics.tpl

```diff
@@ -70,7 +75,10 @@ function to_human(value) {
 {% endjavascript %}
 
 {% javascript %}
-    pubzub.subscribe("erlang/stats/#", function(topic, msg) {
+    pubzub.subscribe("erlang/stats/#", function(topic, msg) {   
+        //print topics to log to see all topics available under erlang/stats/#
+        console.log(topic, msg);
```

Comment:
Can you remove this console.log?
On certain browsers (IE) it is not available and will crash the Javascript.